### PR TITLE
Adding OCP 4.15-ga files

### PR DESCRIPTION
**Pull request by vavuthu**
Signed-off-by: vavuthu [vavuthu@redhat.com](mailto:vavuthu@redhat.com)

### CODE REVIEW

**Review by petr-balogh — APPROVED** [review]

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+RUN:
+  client_version: '4.15-ga'
+DEPLOYMENT:
+  ocp_url_template: "https://mirror.openshift.com/pub/openshift-v4/clients/ocp/{version}/{file_name}-{os_type}-{version}.tar.gz"
+  installer_version: "4.15-ga"
+  terraform_version: "1.0.11"
+  ignition_version: "3.2.0"
+ENV_DATA:
+  vm_template: "rhcos-4.15.0-x86_64-vmware.x86_64"
+  acm_hub_channel: release-2.9
```
Comment (ebenahar) [inline ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml:12]: Should be 2.10

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  acm_version: "2.9"
```
Comment (ebenahar) [inline ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml:13]: need to add `submariner_version: "0.17.0"`

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-minus1-config.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  installer_version: "4.15-ga"
+  terraform_version: "1.0.11"
+  ignition_version: "3.2.0"
+  ocp_channel: "fast"
+  ocp_version_index: -2
+ENV_DATA:
+  # TODO: This we will need to update to update to released version once we
+  # will upload to DCs
+  vm_template: "rhcos-4.15.0-x86_64-vmware.x86_64"
+  acm_hub_channel: release-2.9
```
Comment (ebenahar) [inline ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-minus1-config.yaml:18]: same with acm 2.10 and submariner

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml (hunk 1 above)
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml]: done

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml (hunk 2 above)
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-config.yaml]: done

File: ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-minus1-config.yaml (hunk 3 above)
Comment (vavuthu) [reply ocs_ci/framework/conf/ocp_version/ocp-4.15-ga-minus1-config.yaml]: done

**Review by ebenahar — APPROVED** [review]

**Review by dahorak — APPROVED** [review]

**Pull request merged**